File: src/main/java/com/itechart/demojavakotlin/repository/TicketRepository.java
package com.itechart.demojavakotlin.repository;

import com.itechart.demojavakotlin.entity.TicketEntity;
import com.itechart.demojavakotlin.entity.UserEntity;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

import java.util.List;
import java.util.UUID;

@Repository
public interface TicketRepository extends JpaRepository<TicketEntity, UUID> {
    List<TicketEntity> findByUserId(final UserEntity user);
}

File: kotlinpath/demojavakotlin/src/main/kotlin/by/itechart/demojavakotlin/repository/TicketRepository.kt
package by.itechart.demojavakotlin.repository

import by.itechart.demojavakotlin.entity.TicketEntity
import by.itechart.demojavakotlin.entity.UserEntity
import org.springframework.data.jpa.repository.JpaRepository
import java.util.*

interface TicketRepository : JpaRepository<TicketEntity, UUID> {
    fun findByUserId(user: UserEntity): List<TicketEntity>

}